### PR TITLE
Fix Icon and name not portrayed as expected #2513

### LINKED ISSUE #2513: Icon and name not portrayed as expected
There are several issues with the icon and name of the user on the top menu bar:
* Personal icon is not shown even though it is in the user account
* Icon is slightly cut off on top 
* Icon and name are not in the middle vertically in the menu bar

Screenshot from 2019-07-11 12-46-07

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2513

Changes: Fix Alignment of Icon and name in nav bar

Demo Link: https://pr-2515-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot 2019-07-11 at 8 07 14 PM

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd fixed

File: src/redux/reducers/create.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ const defaultState = {
     language: '',
     image: avatarsIcon,
     imageUrl: '<image_name>',
-    code: '::name <Bot_name>\n::category <Category>\n::language <Language>\n::author <author_name>\n::author_url <author_url>\n::description <description> \n::dynamic_content <Yes/No>\n::developer_privacy_policy <link>\n::image images/<image_name>\n::terms_of_use <link>\n\n\n# replace the following lines:\nquery1|query2|query3...\n!example:<The question that should be shown in public skill displays>\n!expect:<The answer expected for the above example>\nAnswer for the user query\n',
+    code:
+      '::name <Bot_name>\n::category <Category>\n::language <Language>\n::author <author_name>\n::author_url <author_url>\n::description <description> \n::dynamic_content <Yes/No>\n::developer_privacy_policy <link>\n::image images/<image_name>\n::terms_of_use <link>\n\n\n# replace the following lines:\nquery1|query2|query3...\n!example:<The question that should be shown in public skill displays>\n!expect:<The answer expected for the above example>\nAnswer for the user query\n',
```
Comment (akshatnitd) [inline src/redux/reducers/create.js]: I can see a commit  by @Orbiter regarding this change. Are we undoing his change?

Comment (fragm3) [reply src/redux/reducers/create.js]: No, just running prettier, orbiter changes are still there

**Comment by fragm3** [comment]
@akshatnitd @samagragupta made the changes

**Comment by mariobehling** [comment]
Not sure, why it is not showing up as expected after this PR. Have to reopen the issue.

**Comment by mariobehling** [comment]
These changes also changed the "?" dropdown menu. There is now a space appearing.

Screenshot from 2019-07-14 15-44-17

**Comment by mariobehling** [comment]
Added issue here: https://github.com/fossasia/susi.ai/issues/2545

**Pull request merged**